index.html

```diff
@@ -56,21 +57,22 @@
     </div>
   </nav>
     <!-- END nav -->
-    
+
     <!-- <div class="js-fullheight"> -->
     <div class="hero-wrap js-fullheight">
       <div class="overlay"></div>
       <div id="particles-js"></div>
       <div class="container">
         <div class="row no-gutters slider-text align-items-center justify-content-center" data-scrollax-parent="true">
           <div class="col-md-6 ftco-animate text-center" data-scrollax=" properties: { translateY: '70%' }">
-            <h1 class="mb-4" data-scrollax="properties: { translateY: '30%', opacity: 1.6 }">We love to create <strong>Beautiful</strong> websites</h1>
+            <h1 class="mb-4 Long-chang" data-scrollax="properties: { translateY: '30%', opacity: 1.6 }">Manga & Anime
+              Society Kharagpur</h1>
```

Comment:
Can you created a css class with `font-heading` which imports the variable font from the `style.scss` so that when needed to change we change the variable name directly. Rather than changing the name everytime in `index.html`.

Reply:
Ok happy to see review, help me out here so instead of `long-chang` class in h1 heading i need to change it to `font-heading`.
So that when variable `$font-heading` value change, It will updated directly instead of changing its class name.
Right?

Reply:
Ya in h1 you should change the class name to `font-heading`. And that font heading styling is defined in someother `sass(.scss)` file as `font-family: $font-heading`, where this font-heading is a defined sass variable with value the name of the google font. Hope it is clear now? Also please look a bolder and different font like the ones I suggested.